### PR TITLE
AKO Release 1.2.1 changes.

### PR DESCRIPTION
**Pull request by parimanur**

### CODE REVIEW

File: charts/incubator/ako/values.yaml [hunk 1]
```diff
@@ -5,29 +5,21 @@
 replicaCount: 1
 
 image:
-  repository: avinetworks/ako
+  repository: 10.79.172.11:5000/avi-buildops/ako
```
Comment (monotosh-avi) [inline charts/incubator/ako/values.yaml:8]: repo should be avinetwork/ako

File: charts/incubator/ako/values.yaml [hunk 2]
```diff
@@ -36,28 +28,49 @@ configs:
...
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
 
 avicredentials:
-  username: # Avi controller username 
-  password: # Avi controller password
+  username: admin
+  password: orion123
```
Comment (monotosh-avi) [inline charts/incubator/ako/values.yaml:72]: we can remove this and set to empty

File: docs/AKO/openshift/objects.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
+### Openshift Route
```
Comment (monotosh-avi) [inline docs/AKO/openshift/objects.md:1]: If we are creating a separate doc, can we add a link as ref from AKO/objects.md ?

Comment (parimanur) [reply docs/AKO/openshift/objects.md:1]: Done!

**Review by sudswasavi — APPROVED** [review]

**Review by monotosh-avi — APPROVED** [review]

**Pull request merged**